samples/js/fb/find-photos/src/main.js

```diff
@@ -143,6 +144,16 @@ async function getSizes(input): Promise<Map<Struct, string>> {
   return result;
 }
 
+async function getResources(input): Promise<Map<Struct, Struct>> {
+  let result = Promise.resolve(new Map());
+  await input.images.forEach(v => {
+    result = result.then(m => m.set(
+      newStruct('', {width: v.width, height: v.height}),
+      v.source));
```

Comment:
Isn't `v.source` a string? The signature says that this method returns `Map<Struct, Struct>`.